README.md

```diff
@@ -25,6 +25,10 @@ The following table lists the environment variables used for configuration.
 | SQS_RESULT_QUEUE | SQS queue name (not URL or ARN, just the name) from which validation results will arrive |
 | SLACK_WEBHOOK    | Slack channel webhook to which messages will be sent                                     |
 | VERBOSE_OUTPUT   | Set to `TRUE` to activate, all other falues will set this to false                       |
+| RECIPIENTS_DICT  | Stringified JSON object with data provider names as the keys and arrays of email         |
+|                  | addresses as the values                                                                  |
```

Comment:
I think this description needs to be on one line to preserve the table formatting

Reply:
Thanks for the catch - updated the README in new commit.